cns/cnsclient/cnsclient.go

```diff
@@ -266,8 +266,8 @@ func (cnsClient *CNSClient) RequestIPAddress(orchestratorContext []byte) (*cns.I
 	return response, err
 }
 
-// ReleaseIPAddress calls releaseIPAddress on CNS
-func (cnsClient *CNSClient) ReleaseIPAddress(orchestratorContext []byte) error {
+// ReleaseIPAddress calls releaseIPAddress on CNS, ipaddress ex: (10.0.0.1)
+func (cnsClient *CNSClient) ReleaseIPAddress(ipaddress string, orchestratorContext []byte) error {
```

Comment:
would it be better to pass these around as `net.IP` and then string them just in time for use? we could error on parsing at ingest instead of at use if something weird comes in

Reply:
moved it a little bit forward to directly populate the request object to send to the client